docs/backend-requests/resources/session-tokens.mdx

```diff
@@ -11,13 +11,13 @@ When a user is authenticated in your application, a short-lived session token is
 
 Below are the default session claims that Clerk generates for you:
 
-* `azp`: authorized party - the `Origin` header that was included in the original Frontend API request made from the user. Most commonly, it will be the URL of the application, e.g. `https://example.com`.
-* `exp`: expiration time - the time at which this token expires.
-* `iat`: issued at - the time at which this token was issued.
-* `iss`: issuer - the Frontend API URL of the instance, e.g. `https://clerk.example.com`.
-* `nbf`: not before - the time before which this token cannot be used.
+* `azp`: authorized party - the `Origin` header that was included in the original Frontend API request made from the user. Most commonly, it will be the URL of the application (e.g. `https://example.com`).
+* `exp`: expiration time - the time after which the token will expire, as a Unix timestamp. Determined using the **Token lifetime** template setting in the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=jwt-templates).
+* `iat`: issued at - the time at which the token was issued, as a Unix timestamp (e.g. `1516239022`).
+* `iss`: issuer - the Frontend API URL of your instance (e.g. `https://clerk.your-site.com`).
+* `nbf`: not before - the time before which the token is considered invalid, as a Unix timestamp. Determined using the **Allowed Clock Skew** template setting in the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=jwt-templates).
 * `sid`: session ID - the ID of the current session.
-* `sub`: subject - the ID of the current user of the session.
+* `sub`: subject - the ID of the current user of the session (e.g. `user_123`).
 * `orgs`: organizations - the organizations that the user belongs to.
```

Comment:
`orgs` is deprecated, so this won't be returned for new applications.

Reply:
How do we indicate deprecations in documentation currently?

Reply:
Not really sure. I was thinking of either putting that into the text description like `[Deprecated]` or just moving it to another section below, indicating `**Deprecated**`, and having just the `orgs` as an item for now. We could even remove it as it's about instances created more than a year ago.